inference-engine/CMakeLists.txt

```diff
@@ -50,11 +50,14 @@ function(ie_developer_export_targets)
 endfunction()
 
 function(ie_developer_export)
+    export(TARGETS ${OpenVINODeveloperPackageTargets} NAMESPACE IE::
+        APPEND FILE "${CMAKE_BINARY_DIR}/ie_targets_developer.cmake")
```

Comment:
we export library 2 times? with `openvino::` and `IE::` prefixes? why do we need it?

Reply:
Exporting openvino::itt is unnecessary because for all external plugins it should be from IE::.

Reply:
OpenVINODeveloperPackageTargets contains openvino common libraries. We export them in IE:: namespace.